### ISSUE TITLE
Apply button saves a component only once

### ISSUE THREAD

**Issue by poi33** [enonic/app-contentstudio#6761]
Content studio 5.0.4 + next.xp

After saving a part more then once the data is not updated in the inline preview.

Take any part update it multiple times the preview will not update after the first apply of the component

**Comment by alansemenov** [comment]
Fixed in master

**Comment by alansemenov** [comment]
Duplicate of https://github.com/enonic/app-contentstudio/issues/6754

**Comment by alansemenov** [comment]
Reopening. Must be fixed in 5.0 branch
